scrabble.js

```diff
@@ -1,8 +1,204 @@
-var Scrabble = function() {};
 
-// YOUR CODE HERE
-Scrabble.prototype.helloWorld = function() {
-  return 'hello world!';
-};
+  var prompt = require('prompt');
+  var request = require('request');
 
+// SCRABBLE
+  Scrabble = function() {
+    this._points = {
+      A: 1,
+      B: 3,
+      C: 3,
+      D: 2,
+      E: 1,
+      F: 4,
+      G: 2,
+      H: 4,
+      I: 1,
+      J: 8,
+      K: 5,
+      L: 1,
+      M: 3,
+      N: 1,
+      O: 1,
+      P: 3,
+      Q: 10,
+      R: 1,
+      S: 1,
+      T: 1,
+      U: 1,
+      V: 4,
+      W: 4,
+      X: 8,
+      Y: 4,
+      Z: 10
+    };
+  };
+  Scrabble.prototype.score = function(word){
+    var score = 0;
+    var tiles = 0;
+
+    var wordArray = word.toUpperCase().split("");
+
+    for(var char of wordArray){
+      score += this._points[char];
+
+      tiles += 1;
+    };
+
+    if(tiles == 7){
+      score += 50;
+    };
+    return score;
+  };
+
+  Scrabble.prototype.highestScoreFrom = function(arrayOfWords){
+
+    var maxWord = [arrayOfWords[0], this.score(arrayOfWords[0])];
+    var currentScore = this.score(arrayOfWords[0]);
```

Comment:
Nitpick - It's not efficient to call the score function twice on the same word.

Could just do `currentScore = maxWord[1]` for example